examples/ant_world/ant_world.cc

```diff
@@ -112,7 +112,7 @@ int main(int argc, char **argv)
                                     4096,
                                     GL_COMPRESSED_RGB);
     } else {
-        renderer.getWorld().load("../../libantworld/world5000_gray.bin",
+        renderer.getWorld().load("../../include/antworld/world5000_gray.bin",
```

Comment:
it seems a little odd that data is in the include directory

Reply:
True. Maybe we should make a ``resources`` folder for these misc files we have floating around.

Reply:
:+1: